apps/builder/app/canvas/shared/styles.ts

```diff
@@ -324,7 +323,22 @@ export const GlobalStyles = ({ params }: { params: Params }) => {
 
   useLayoutEffect(() => {
     presetSheet.clear();
-    addPresetRules(presetSheet, metas);
+    presetSheet.addMediaRule("presets");
+    for (const [component, meta] of metas) {
+      for (const [tag, styles] of Object.entries(meta.presetStyle ?? {})) {
+        const rule = presetSheet.addNestingRule(
+          `${tag}:where([data-ws-component="${component}"])`
+        );
+        for (const declaration of styles) {
+          rule.setDeclaration({
+            breakpoint: "presets",
```

Comment:
weird to see breakpoint name presets, since they are usually  different

Reply:
It's just static identifier to place all styles within single media rule.